### PR TITLE
Fix type stubs

### PR DESCRIPTION
**Pull request by arvidn**
I also added a CI job to ensure the generated file is up-to-date.

### CODE REVIEW

File: wheel/chia_rs.pyi [hunk 1]
```diff
@@ -31,7 +31,7 @@ class LazyNode:
     def atom() -> bytes: ...
 
 def serialized_length(program: bytes) -> int: ...
-def tree_hash(program: bytes) -> bytes32: ...
+def tree_hash(program: ReadableBuffer) -> bytes32: ...
```
Comment (richardkiss) [inline wheel/chia_rs.pyi:34]: I wasn't able to find `ReadableBuffer` declared or imported. Does this work with mypy?

**Review by richardkiss — APPROVED** [review]
Seems good, except for question about `ReadableBuffer`.

File: wheel/chia_rs.pyi (hunk 1 above)
Comment (arvidn) [reply wheel/chia_rs.pyi:34]: it's on line 9 in this file. It's a poor-man's definition of a python buffer.

Comment (arvidn) [reply wheel/chia_rs.pyi:34]: the reason this changed here was that I forgot to regenerate this file when merging this change

Comment (richardkiss) [reply wheel/chia_rs.pyi:34]: Ah, got it, sorry, not sure how I missed it.

**Pull request merged**